* on interix, $firstmakefile may not be 'makefile', since the filesystem
  may be case insensitive, and perl will wrongly delete Makefile.

--- perl-5.8.8/Configure
+++ perl-5.8.8/Configure
@@ -4528,7 +4528,13 @@
 
 : Which makefile gets called first.  This is used by make depend.
 case "$firstmakefile" in
-'') firstmakefile='makefile';;
+'') 
+	case "$osname" in
+	# WARNING: maybe case insensitive!! So assume we have GNU make
+	interix) firstmakefile='GNUmakefile' ;;
+	*) firstmakefile='makefile' ;;
+	esac
+	;;
 esac
 
 case "$ccflags" in
